### PR TITLE
Corda 4.8.11 patch release

### PR DESCRIPTION
**Pull request by amiegrace12**
Preview built here: https://amie.preview.docs.r3.com/en/platform/corda/4.8/enterprise/release-notes-enterprise.html#corda-enterprise-edition-4811-release-notes

### CODE REVIEW

File: content/en/platform/corda/4.8/enterprise/release-notes-enterprise.md [hunk 1]
```diff
@@ -16,6 +16,58 @@ weight: 10
...
+
+* When a notary worker is shut down, message ID cleanup is now performed as the last shutdown activity, rather than the first; this prevents a situation where the notary worker might still appear to be part of the notary cluster and receiving client traffic while shutting down.
+
+* Flow checkpoint dumps now include a `status` field which shows the status of the flow; in particular, whether it is hospitalized or not.
+
+* Debug logging of the Artemis server has been added.
+
+* During recovery from a transport layer connection break in peer-to-peer connectivity, a workaround to a bug in the Artemis message broker would only be taken during the first break in connectivity. This lead to a rare failure to re-establish connectivity between two peers until the node was restarted. The workaround is now taken on every loss of connectivity, and thus peer-to-peer connectivity should now always be re-established without operator intervention.
```
Comment (nadinequinr3) [inline content/en/platform/corda/4.8/enterprise/release-notes-enterprise.md:37]:
```suggestion
* During recovery from a transport-layer connection break in peer-to-peer connectivity, a workaround to a bug in the Artemis message broker was only performed during the first break in connectivity. This led to a rare failure to re-establish connectivity between two peers until the node was restarted. The workaround is now performed on every loss of connectivity, and thus peer-to-peer connectivity should now always be re-established without operator intervention.
```
not sure about the "taken"/"performed "

File: content/en/platform/corda/4.8/enterprise/release-notes-enterprise.md [hunk 2]
```diff
@@ -16,6 +16,58 @@ weight: 10
...
+* Previously, when loading checkpoints, the only log messages recorded were at the end of the process, recording the total number of checkpoints loaded.
+
+  Now, the following additional logging has been added:
+
+  * Checkpoints: Logging has been added for the two types of checkpoints - runnable and paused flows - being loaded; log messages show the number of checkpoints loaded every 30 seconds until all checkpoints have been loaded.
+
+  * Finished flows: Log messages now show the number of finished flows.
+
+  For example:
+  ```
```
Comment (nadinequinr3) [inline content/en/platform/corda/4.8/enterprise/release-notes-enterprise.md:56]: the colour of the text in the preview seems off..
Maybe try:
```suggestion
  ```shell
```

File: content/en/platform/corda/4.8/enterprise/release-notes-enterprise.md [hunk 3]
```diff
@@ -16,6 +16,58 @@ weight: 10
...
+  [INFO ] 2023-02-03T17:00:28,437Z [main] statemachine.MultiThreadedStateMachineManager. - Loaded 9001 finished flows {}
+  [INFO ] 2023-02-03T17:00:43,606Z [main] statemachine.MultiThreadedStateMachineManager. - Loaded 24001 finished flows {}
+  [INFO ] 2023-02-03T17:00:46,650Z [main] statemachine.MultiThreadedStateMachineManager. - Number of finished flows : 27485 {}
+  ```
+
+* A fix for cache eviction has been applied where an issue resulted in an incorrect contract verification status while a database transaction was in progress during contract verification.
+
+* The default SSL handshake timeout for inbound connections has been increased to 60 seconds. If during SSL handshake, certificate revocation lists (CRLs) take a long time to download, or are unreachable, then this 60 seconds gives the node enough time to establish the connection if `crlCheckSoftFail` is enabled.
+
+* Improved compatibility when using the performance test suite from Apple silicon Macs.
```
Comment (nadinequinr3) [inline content/en/platform/corda/4.8/enterprise/release-notes-enterprise.md:69]:
```suggestion
* Compatibility when using the performance test suite from Apple silicon Macs has been improved.
```

**Review by nadinequinr3 — APPROVED** [review]

**Review by yarivus — COMMENTED** [review]
thx @amiegrace12

**Pull request merged**